exonum/src/runtime/dispatcher/migration_tests.rs

```diff
@@ -203,6 +219,23 @@ impl Rig {
         }
     }
 
+    fn new_with_db(db: Arc<TemporaryDB>) -> Self {
```

Comment:
Nit: `Rig::new()` can call `Self::new_with_db` in order to avoid code duplication. Also, `with_db` would be a more idiomatic name, IMO.